enhavo/dashboard-bundle/0.8/config/packages/enhavo_dashboard.yaml

```diff
@@ -0,0 +1,2 @@
+imports:
+    - { resource: '@EnhavoDashboardBundle/Resources/config/app/config.yml' }
```

Comment:
Shouldn't the bundle import this file?

Reply:
A lot of configuration files in enhavo will overwrite other bundle settings. So i think they can't be just "imported" by the bundle. The symfony documentation said, to load these kinds of configuration over ``PrependExtensionInterface``.  That is an ongoing work in enhavo and will be used for ``0.9``. For enhavo ``0.8`` they are still loaded over the configuration

Reply:
> A lot of configuration files in enhavo will overwrite other bundle settings

That is not really a preferred thing.. In fact. I would suggest to try different solution.